Build fails on mac

The issue seems to be related to the presence of a system python and/or VTK libraries it installed.

```
-- Checking for module 'libgphoto2'
--   Package 'libgphoto2' not found
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- To enable PlantUML support, set PLANTUML_JAR environment variable or pass -DPLANTUML_JAR=<filepath> option to cmake
-- Found PythonInterp: /Users/fedorov/local/builds/Slicer4-Debug/python-install/bin/SlicerPython (found suitable version "2.7.10", minimum required is "2.7")
-- Found PythonLibs: /usr/lib/libpython2.7.dylib (found suitable exact version "2.7.10")
-- Could NOT find PythonInterp: Found unsuitable version "2.7.10", but required is at least "3.4" (found /usr/bin/python)
-- Could NOT find PythonInterp: Found unsuitable version "2.7.10", but required is at least "3.2" (found /usr/bin/python)
-- Found JNI: /System/Library/Frameworks/JavaVM.framework
-- Found Matlab: /Applications/MATLAB_R2013b.app/bin/mex
CMake Error at /Users/fedorov/Library/Enthought/Canopy_64bit/User/lib/cmake/vtk-6.2/VTKTargets.cmake:1445 (message):
  The imported target "vtkWrapPython" references the file

     "/Users/fedorov/Library/Enthought/Canopy_64bit/User/bin/vtkWrapPython-6.2"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/Users/fedorov/Library/Enthought/Canopy_64bit/User/lib/cmake/vtk-6.2/VTKTargets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /Users/fedorov/Library/Enthought/Canopy_64bit/User/lib/cmake/vtk-6.2/VTKConfig.cmake:60 (include)
  cmake/OpenCVDetectVTK.cmake:6 (find_package)
  CMakeLists.txt:584 (include)


-- Configuring incomplete, errors occurred!
```

--- a/SuperBuild/External_OpenCV.cmake
+++ b/SuperBuild/External_OpenCV.cmake
@@ -82,6 +82,8 @@ if(NOT DEFINED OpenCV_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
       -DBUILD_opencv_python2:BOOL=OFF
       -DBUILD_opencv_ts:BOOL=OFF
       -DBUILD_opencv_world:BOOL=OFF
+      # Disable VTK: not used, and is causing problems
+      -DWITH_VTK:BOOL=OFF
     DEPENDS
       ${${proj}_DEPENDENCIES}
     )

